File: src/SOP8/광양/EgovLogin/base-model/src/main/java/egovframework/base/model/common/team/Regular.java
package egovframework.base.model.common.team;

import lombok.*;
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Regular {
	public enum Fields { rgl_sn, team_name, parnts_sn, site_sn }

	private int rgl_sn;

	@NonNull
	@Builder.Default
	private String team_name = "";

	private Integer parnts_sn;

	private Integer site_sn;

	public static final String autoIncreaseField = "rgl_sn";
	public static final String tableName = "co_team_rgl";
	public static final String[] primaryKeys = { "rgl_sn" };

}

File: src/SOP8/Base/Base.Model/Common/Team/Regular.cs
using System;
using dnsDapperDBUtil.DataAccessLayer.IDAL;

namespace Base.Model.Common.Team
{
	public class Regular : Table
	{
		public enum Fields { rgl_sn, team_name, parnts_sn, site_sn };
		public enum WriteFields { team_name, parnts_sn, site_sn };

		public int rgl_sn { get; set; }
		public string team_name { get; set; }
		public int? parnts_sn { get; set; }
		public int? site_sn { get; set; }

		public static string TableName { get { return "co_team_rgl"; } }

		public override string GetTableName()
		{
			return TableName;
		}

		public override string GetPrimaryCondition()
		{
			return string.Format("{0} = {1}", Fields.rgl_sn, rgl_sn);
		}

		public override Type GetFieldType()
		{
			return typeof(Fields);
		}

		public override Type GetWriteFieldType()
		{
			return typeof(WriteFields);
		}

		public void FromCopy(Regular obj)
		{
			this.rgl_sn = obj.rgl_sn;
			this.team_name = obj.team_name;
			this.parnts_sn = obj.parnts_sn;
			this.site_sn = obj.site_sn;
		}
	}
}
